gateway/src/service.rs

```diff
@@ -652,7 +663,38 @@ pub mod tests {
 
         // If recreated by the same user
         assert!(matches!(
-            svc.create_project(matrix, neo).await,
+            svc.create_project(matrix.clone(), neo, false).await,
+            Ok(Project::Creating(_))
+        ));
+
+        // If running project is created a different user which is admin
```

Comment:
```suggestion
        // If running project is created by a different user which is not admin
```

Reply:
Ah, woops, this was supposed to test that if an admin tried to `project new` with the name of running project, it would still return `ProjectAlreadyExists`. It's a bit redundant, perhaps... 

There is an assertion above this that tests if a different user that isn't admin tries to create a project with a taken name.

Reply:
Haha, okay. Maybe we can remove this one then?

Reply:
Yeah, I'll do that.